test/DynamoCoreWpfTests/SerializationTests.cs

```diff
@@ -685,6 +712,39 @@ private string ConvertCurrentWorkspaceViewToNonGuidJsonAndSave(DynamoViewModel v
             return json;
         }
 
+        private static void ConvertCurrentWorkspaceToDesignScriptAndSave(string filePathBase, DynamoModel currentDynamoModel)
```

Comment:
This was originally removed because the design script files are already generated in the DynamoCore serialization tests.  Should we remove it from DynamoCoreTests and generate .ds files for all DynamoCoreWpfTests instead of duplicating the code?  See [this comment](https://github.com/alfarok/Dynamo/blob/aac9b9eb1d20ea47592fd7f261525b9dd7b69e1d/test/DynamoCoreWpfTests/SerializationTests.cs#L540)

Reply:
One alternative is to make `ConvertCurrentWorkspaceToDesignScriptAndSave`  a public static method in DynamoCoreTests and call it when the .data files are being generated

Reply:
is this duplicate of what is there in core?

Reply:
@ramramps yes but I can't access it directly.  Another option is to move that method to `serializationTestUtils` which contains test helpers shared between the two.

Reply:
sure that makes sense.